scrapers/wordpress.js

```diff
@@ -1,62 +1,92 @@
-/* Usage : wordpress.js <blog_url>
- */
+const BaseLoklakScraper = require('./base');
 
-var request = require("request");
-var cheerio = require("cheerio");
 
-var url = null;
+const cheerio = require('cheerio');
+const request = require('request-promise-native');
+const Rx = require('rxjs/Rx');
 
-// blog url must be provided as a command line argument
+const USER_AGENT = "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 " +
```

Comment:
could you add this line in BaseScraper and use it here, this way it can be used in every scraper